app/views/employer_profiles/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>My profile</h1>
+
+  <p>First name: <%= current_employer.first_name %></p>
```

Comment:
This should be using @employer instead of current_employer as you are setting @employer in the controller. Same for the rest of the file